### PR TITLE
Added a feature to save logs to local storage

### PR DESCRIPTION
**Pull request by ALEX5402**
- Added export logs to local storage [1664](https://github.com/tiann/KernelSU/issues/1664)

### CODE REVIEW

File: manager/app/build.gradle.kts [hunk 1]
```diff
@@ -23,8 +23,8 @@ android {
 
     buildTypes {
         release {
-            isMinifyEnabled = true
-            isShrinkResources = true
+            isMinifyEnabled = false
```
Comment (tiann) [inline manager/app/build.gradle.kts:26]: please don't touch this

File: manager/app/src/main/java/me/weishu/kernelsu/ui/screen/Settings.kt [hunk 2]
```diff
@@ -151,6 +158,49 @@ fun SettingScreen(navigator: DestinationsNavigator) {
                 enableWebDebugging = it
             }
 
+            ListItem(
+                leadingContent = {
+                    Icon(
+                        Icons.Filled.Save,
+                        stringResource(id = R.string.save_log)
```
Comment (tiann) [inline manager/app/src/main/java/me/weishu/kernelsu/ui/screen/Settings.kt:165]: We already have "send log". It is better to merge these two.

**Review by tiann — CHANGES_REQUESTED** [review]

File: manager/app/build.gradle.kts (hunk 1 above)
Comment (ALEX5402) [reply manager/app/build.gradle.kts:26]: ok let me fix it i forgot to change this after test

File: manager/app/src/main/java/me/weishu/kernelsu/ui/screen/Settings.kt (hunk 2 above)
Comment (ALEX5402) [reply manager/app/src/main/java/me/weishu/kernelsu/ui/screen/Settings.kt:165]: ok got it

Comment (ALEX5402) [reply manager/app/src/main/java/me/weishu/kernelsu/ui/screen/Settings.kt:165]: But having this feature inside this app makes it more convenient and then user don't need to install any other apps just to do this simple thing

**Comment by tiann** [comment]
we don't need two entries in settings, "save log" and "send log", it's strange, we only need one entry, when clicked, you can prompt user to choose

**Comment by ALEX5402** [comment]
> we don't need two entries in settings, "save log" and "send log", it's strange, we only need one entry, when clicked, you can prompt user to choose

ok got it 

> choose

can i add a alert dialog there to choose user what to do ?

**Comment by tiann** [comment]
> can i add a alert dialog there to choose user what to do ?

that's ok

**Comment by ALEX5402** [comment]
> > can i add a alert dialog there to choose user what to do ?
> 
> that's ok

done sir please check this [commit](https://github.com/ALEX5402/KernelSU/commit/220ef324d0594bb3d52db8d0903fec2e39b9014d)

**Review by tiann — APPROVED** [review]

**Pull request merged**